README.rst

```diff
@@ -6,13 +6,166 @@ App for tracking sprint commitments, vacations and spillovers.
+
+1. Upload spillovers.
+    This uploads all spillovers to the `GOOGLE_SPILLOVER_SPREADSHEET`. The following rows are filled in the spreadsheet:
+
+    a) Ticket
+        The key of the ticket.
+    b) Status
+        The status of the ticket at the moment of ending the sprint.
+    c) Sprints
+        The active sprint (the one that is currently being ended).
+    d) Assignee
+        The assignee, for whom the spillover is being counted.
+    e) Reviewer 1
+    f) Reviewer 2
+    g) Reporter
+    h) Story Points
+    i) Estimated time
+        The initial estimation of the ticket (in hours).
+    j) Remaining time
+        The remaining time for the ticket (in hours).
+    k) Reason for the spillover
+        The reason of the spillover is retrieved from the comments made within the active sprint. The assignees should provide it with a comment matching the following regexp: ```[~{JIRA_BOT_USERNAME}\]: <spillover>(.*)<\/spillover>```. In case of multiple occurrences of comments matching this regexp, only the last one is taken into account. In case of no occurrences of such comments, the Jira bot will create a comment defined in `SPILLOVER_REMINDER_MESSAGE`.
+
+    If the users have achieved the clean sprint (without spillovers), they can post some hints on the ticket with the `SPRINT_MEETINGS_TICKET` name by adding a comment matching the spillover reason regexp (provided above). In case of no such comment, they will be reminded on the ticket with `SPILLOVER_CLEAN_HINTS_MESSAGE` comment. It's possible to disable the pings for specific users by adding them to `SPILLOVER_CLEAN_SPRINT_IGNORED_USERS` (this can be useful for people that are members of multiple cells, as they will be pinged on each cell-specific ticket).
+2. Upload commitments.
+    The `goal` of each user from the dashboard is uploaded to the cell-specific commitments sheet of the `GOOGLE_SPILLOVER_SPREADSHEET`.
+3. Move archived issues out of the active sprint.
+    There has been a bug before that disallowed completing the sprint if it had archived issues, so we're moving all of them out of the active sprint.
+4. Close the active sprint.
+5. Move issues from the closed sprint to the next one.
+6. Open the next sprint.
+7. Create role-specific tasks for the future sprint.
+    The assignees for these tickets are retrieved from the `GOOGLE_ROTATIONS_RANGE` defined within `GOOGLE_ROTATIONS_SPREADSHEET`. The format of this document is the following:
+
+    a) First column contains sprint number (you can create multiple role tasks for one week by dividing sprint into parts, e.g. `Sprint 100a, Sprint 100b` - each in a separate row).
+    b) Next columns' headers contain role names prefixed by the full cell name (e.g. `Cell_1 FF`) and their fields contain assignees for the tickets.
```

Comment:
@Agrendalath This doesn't mention the "Date" column. Probably not that important, so up to you whether you think it's worth adding some info about that column or not :)

Reply:
It's just omitted, but I added a short note about it.

Reply:
@Agrendalath Thanks :) Since this describes the format of the rotations spreadsheet, I think it would be worth listing it as the second item:

```
    a) First column contains the sprint number (you can create multiple role tasks for one week by dividing sprint into parts, e.g. `Sprint 100a, Sprint 100b` - each in a separate row).
    b) The second column contains the start date for a (partial) sprint. It is not used for the purpose of creating role-specific tasks.
    c) Next columns' headers contain role names prefixed by the full cell name (e.g. `Cell_1 FF`) and their fields contain assignees for the tickets.
```